content/learn/migration-guides/0.10-0.11/_index.md

```diff
@@ -0,0 +1,830 @@
+
+- Use `2> output.log` on the command line to save `stderr` to a file
+
+### [Make `WorldQuery` meta types unnameable](https://github.com/bevyengine/bevy/pull/7964)
+
+<div class="migration-guide-area-tags">
+    <div class="migration-guide-area-tag">ECS</div>
+</div>
+
+The `State` and `Fetch` types for types created using `#[derive(WorldQuery)]` are now unnameable. If you need to refer to them, use the syntax `<T as WorldQuery>::State`, `<T as WorldQuery>::Fetch`.
+
+### [Increase type safety and clarity for change detection](https://github.com/bevyengine/bevy/pull/7905)
+
+<div class="migration-guide-area-tags">
+    <div class="migration-guide-area-tag">ECS</div>
+</div>
+
+The engine now uses the type `Tick` for dealing with change ticks, instead of `u32`. Any code that interfaced with engine internals will need to be updated, including:
+
+- Manual implementers of the traits `SystemParam`, `WorldQuery`, `DetectChanges`, and `DetectChangesMut`.
+- The methods `World::change_tick` and `read_change_tick`.
+- `System::set_last_change_tick` and `get_last_change_tick`. Also, these methods have been renamed to `set_last_run` and `get_last_run`, respectively.
+- The methods `SystemChangeTick::change_tick` and `last_change_tick`. These methods have been renamed to `this_run` and `last_run`, respectively.
+- The method `Tick::set_changed`, which has been renamed to just `set`.
+
+### [Remove ChangeTrackers](https://github.com/bevyengine/bevy/pull/7902)
+
+<div class="migration-guide-area-tags">
+    <div class="migration-guide-area-tag">ECS</div>
+</div>
+
+`ChangeTrackers` has been removed .Use `Ref<T>` queries instead.
+
+### [Check for conflicting accesses in `assert_is_system`](https://github.com/bevyengine/bevy/pull/8154)
+
+<div class="migration-guide-area-tags">
+    <div class="migration-guide-area-tag">ECS</div>
+</div>
+
+The functions `assert_is_system` and `assert_is_read_only_system` (in `bevy_ecs::system`) now panic if the passed system has invalid world accesses. Any tests that called this function on a system with invalid accesses will now fail. Either fix the system’s conflicting accesses, or specify that the test is meant to fail:
+
+- For regular tests (that is, functions annotated with `#[test]`), add the `#[should_panic]` attribute to the function.
+- For documentation tests, add `should_panic` to the start of the code block: ` ```should_panic`
+
+### [`Or<T>` should be a new type of `PhantomData<T>`](https://github.com/bevyengine/bevy/pull/8212)
+
+<div class="migration-guide-area-tags">
+    <div class="migration-guide-area-tag">ECS</div>
+</div>
+
+`Or<T>` is just used as a type annotation and shouldn’t be constructed.
+
+### [Remove base set error variants of `ScheduleBuildError`](https://github.com/bevyengine/bevy/pull/8269)
+
+<div class="migration-guide-area-tags">
+    <div class="migration-guide-area-tag">ECS</div>
+</div>
+
+Remove the now unused variants of `ScheduleBuildError`

```

Comment:
There's not really anything to migrate, some error variants don't happen anymore so you need to remove code that handled them. This can be rephrased I guess but there's nothing to migrate too. I'm not sure how to phrase it to make that clear.

Reply:
So... what's the error happening in the same conditions then? Or was that error scenario completely removed?

Reply:
The PR description says those errors can't happen anymore. I think I just need to figure out how to phrase that better. The PR didn't have a migration guide but it's still a breaking change so I'd like to at least warn users of this.

Reply:
Looking at the PR again, it's only base sets related variants that have been removed since base sets have been removed. I rephrased the guide to reflect that more clearly